doc/source/examples/incompressible-flow/2d-lid‐driven-cavity-flow/lid‐driven-cavity-flow.rst

```diff
@@ -0,0 +1,306 @@
+==================================
+2D Lid-Driven Cavity Flow
+==================================
+
+This is the first Lethe example. It showcases a classical fluid mechanics problem, the lid-driven cavity. This example also introduces the concept of parameter files to parametrize Lethe simulations.
+
+Features used in this example
+----------------------------------
+- Solver : ``gls_navier_stokes_2d``, ``gd_navier_stokes_2d`` (with Q2-Q1)
+- Steady-state problem
+- Setting initial conditions
+- Adjoint time-stepping for steady-state problems
+
+
+Location of the examples
+------------------------
+- Base case (Re=400) : ``/examples/incompressible_flows/cavity.prm``
+- Re=7500 : ``/examples/incompressible_flows/Reynolds_7500/cavity.prm``
```

Comment:
Path could be a link to the mentioned file on GitHub

Reply:
Good question. I'm not sure which is better here. I guess specifying the path will enable users to find quickly the example they are looking for and run it. Pointing to the github repo would have a similar impact, but would be a bit more indirect I guess.